### PR TITLE
refactoring

### PR DESCRIPTION
**Pull request by daiki44**
## What?
- create constants dir
- add transfer success message

### CODE REVIEW

File: src/components/Base/Message.js [hunk 1]
```diff
@@ -8,7 +7,7 @@ export default props => {
       <style jsx>{`
         p {
           margin: 1rem;
-          color: ${TEXT_ERROR};
+          color: ${props.color};
```
Comment (tkmct) [inline src/components/Base/Message.js:10]: what about putting default color in case `props.color` is undefined?

File: src/store/deposit.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ import clientWrapper from '../client'
 import { utils } from 'ethers'
 import JSBI from 'jsbi'
 // import { PETHContract } from '../contracts/PETHContract'
-// import { TOKEN_LIST } from '../tokens'
+// import { TOKEN_LIST } from '../constants/tokens'
```
Comment (tkmct) [inline src/store/deposit.js:6]: Let's remove this line

File: src/components/Base/Message.js (hunk 1 above)
Comment (daiki44) [reply src/components/Base/Message.js]: fixed!
https://github.com/cryptoeconomicslab/gazelle-wallet/pull/149/commits/73726b187b27c7a56b2e8de7ec6393d8f8360b08

File: src/components/Base/Message.js [hunk 3]
```diff
@@ -1,15 +1,21 @@
...
+const Message = ({ children, color }) => {
   return (
     <p>
-      {props.children}
+      {children}
       <style jsx>{`
         p {
           margin: 1rem;
-          color: ${props.color};
+          color: ${color};
```
Comment (tkmct) [inline src/components/Base/Message.js:11]: you can just `${color || MAIN}`

**Pull request merged**